.github/workflows/terraform-validate.yml

```diff
@@ -0,0 +1,13 @@
+name: Terraform Validate
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  sync:
```

Comment:
maybe the job should be called validate instead of sync?

Reply:
You are indeed correct